### DISCUSSION TITLE
metadata

### DISCUSSION THREAD

**Discussion by darkobas2** [deepfence/ThreatMapper#361] [Q&A]
Hi

I was searching and couldnt find information on how to edit/configure the following  metadata information:
Cloud Provider
Cloud Region
Cloud Label
any hints ?

**Comment by ramanan-ravi** [comment]
ThreatMapper sensors calls cloud metadata api service provided by different cloud providers to get cloud specific metadata.
This is read only.
If labels are changed in cloud provider's console, it will reflect here.
